tests/integration/fix_non_normalized_codes.t

```diff
@@ -104,10 +104,10 @@ is_deeply(
 	[
 		# removed product_broken_code
 		"Removed broken-123",
-		# removed product with existing id
-		"Removed 0000012345670 as duplicate of 12345670",
 		# product_non_normalized_code moved
 		"Moved 0000012345678 to 12345678",
+		# removed product with existing id
```

Comment:
This is a bug that is not related to your PR, could you remove this change?
The bug is fixed in this PR: https://github.com/openfoodfacts/openfoodfacts-server/pull/8384